### PR TITLE
Document ignored arguments section in java dumps

### PR DESCRIPTION
**Pull request by keithc-ca**
As [requested](https://github.com/eclipse-openj9/openj9/pull/13579#issuecomment-975461319).

Also, clean up:
* collapse multiple blank lines
* trim trailing whitespace
* expand tabs

### CODE REVIEW

**Comment by pshipton** [comment]
Do you think it's worth adding this to the 0.30 release notes?

**Comment by keithc-ca** [comment]
> Do you think it's worth adding this to the 0.30 release notes?

Yes, I think so. I'll put that together.

**Comment by pshipton** [comment]
We'll also need to create 0.30 release notes in OpenJ9 that list this item, see https://github.com/eclipse-openj9/openj9/tree/master/doc/release-notes

**Comment by keithc-ca** [comment]
See #13976.

**Comment by keithc-ca** [comment]
Rebased and added to 0.30.0 release notes.

**Comment by keithc-ca** [comment]
@pshipton Can you have another look, please?

File: docs/version0.30.md [hunk 1]
```diff
@@ -41,6 +41,19 @@ To learn more about support for OpenJ9 releases, including OpenJDK levels and pl
 
 The format of classes that are stored in the shared classes cache is changed, which causes the JVM to create a new shared classes cache rather than re-creating or reusing an existing cache. To save space, you can remove all existing shared caches unless they are in use by an earlier release. For more information about the `-Xshareclasses` option, including the `destroy` options that you can use to remove caches, see [`-Xshareclasses`](xshareclasses.md).
 
+### Ignored options captured in java dumps
```
Comment (pshipton) [inline docs/version0.30.md:44]: Pls add this to the table of contents above.

Comment (keithc-ca) [reply docs/version0.30.md:45]: Done.

**Review by pshipton — APPROVED** [review]

**Comment by pshipton** [comment]
We can't test or publish new docs atm due to UNB machine issues. I will merge and we can fix any undiscovered issues later.

**Comment by pshipton** [comment]
The new docs are published now. https://eclipse-openj9.github.io/openj9-docs/

I think there should be a link from the release notes to https://eclipse-openj9.github.io/openj9-docs/dump_javadump/

**Pull request merged**